codegen/render/kube_crud_test.go

```diff
@@ -69,7 +69,7 @@ func mustCrud(ctx context.Context, clientSet Clientset, paint *Paint) {
 	Expect(written.Spec).To(Equal(paint.Spec))
 
 	status := PaintStatus{
-		ObservedGeneration: written.Generation,
+		ObservedGeneration: uint32(written.Generation),
```

Comment:
is this correct? no longer needed, correct?

Reply:
correct, i restored the int64's